mavros/src/lib/ftf_frame_conversions.cpp

```diff
@@ -91,22 +91,23 @@ Covariance6d transform_static_frame(const Covariance6d &cov, const StaticTF tran
 	EigenMapCovariance6d cov_out(cov_out_.data());
 
 	Eigen::MatrixXd Affine6dTf(6,6);
-	Eigen::Matrix3d R = Eigen::MatrixXd::Identity(3,3);
+	Eigen::Matrix3d _R = NED_ENU_Q.normalized().toRotationMatrix();
+	Eigen::Matrix3d R_ = AIRCRAFT_BASELINK_Q.normalized().toRotationMatrix();
```

Comment:
Should be `static const` since that R do not change.
`_` in prefix/suffix is bad! Call it `R_NE`, `R_AB`.

Is `normalized()` required?
I tested:
```cpp
auto q1 = NED_ENU_Q;
auto q2 = NED_ENU_Q.normalized();
```

```
q1: [0.000000 0.707107 0.707107 0.000000]
q2: [0.000000 0.707107 0.707107 0.000000]
```